### ISSUE TITLE
Chiti_2010 fails QAQC

### ISSUE THREAD

**Issue by jb388** [International-Soil-Radiocarbon-Database/ISRaD#164]
@alkalifly Chiti_2010 is failing QAQC. I'm tagging you because you were the one who commented on the most recent change to the file (didn't find any record of it in the info.israd account).

Here's the error report from QAQC:
 profile tab...
	WARNING: column name mismatch template: pro_lc_phenology pro_lc_leaf_type pro_lc_photosyn_path

Obviously it's fine to add these columns to the master template/info files if you'd like them added, but that needs to be done before the Chitit_2010 template is ingested. 

To move forward on this, we need to:
1) Make a new dev branch
2) Add the proposed columns to BOTH the master template and the template info files
3) Run build on new dev branch
4) Submit a pull request to have dev merged to master (assuming build goes OK)
5) Resubmit the Chiti_2010 file w/ passing QAQC txt file (you can tag me as reviewer)

Let me know if you have any questions or if you had nothing to do with this...

**Comment by jb388** [comment]
Oh, and let me know if you need me to email you the file in question.

**Comment by alkalifly** [comment]
Hi Jeff,

It seems the problem is not with the Chiti_2010 file, but instead is with master template you committed earlier today (perhaps already yesterday, in Germany). 

I made commit d84a122795166c96c1f12229cdee30d1717d1c1e on January 18, which updated the master template and info files to add those pro_lc columns that we had discussed. When I modified Chiti_2010, a few days later, I was using the new template file, so it passed QAQC (I only committed after going through the whole build script successfully).

Then 12 hours ago, you made commit 032f3ca6d986ea29ae3d5bd709d29c57c5e0b980 which replaced the master template file with a version that did not include the extra columns. And of course, with those additional columns no longer in the master template, any template that contains them will begin to fail QAQC.

I'm not sure how this file got out of sync. I added the new columns to the master 14 days ago, but I imagine you have pulled since then. So how did your master template get out of sync as to not have the extra columns?

The solution depends on whether you actually made any intentional changes to the master template in your recent commit, or whether it simply reverted to a version without the pro_lc columns. If the latter, then we can just switch it back to a version that has those columns. If the former, then we have to merge your changes with mine into a new version.

**Comment by jb388** [comment]
@alkalifly 
OK, I figured your intentions were good : ) I really don't think it was just my version of the template, though: I had been pulling up-to-date versions of dev every two days or so since I was in CA. 

I guess it's a possibility that something strange happened when merging dev with master. The branches had diverged a bit so I had to do some conflict resolution. The odd thing is that some of the other columns you added made it in just fine. 

Anyway, I can add the columns. The only reason I didn't is because I wasn't exactly sure what to write in the description section of the template info file. If you want to weigh in on it, or make the necessary changes afterwards, that would be helpful.
